plugins/package/src/main/kotlin/net/corda/cli/plugins/packaging/CreateCpb.kt

```diff
@@ -0,0 +1,130 @@
+package net.corda.cli.plugins.packaging
+
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.UUID
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.nio.file.StandardOpenOption.WRITE
+import java.nio.file.StandardOpenOption.READ
+import java.util.jar.Attributes
+import java.util.jar.JarEntry
+import java.util.jar.JarOutputStream
+import java.util.jar.Manifest
+import net.corda.cli.plugins.packaging.signing.CpxSigner
+
+@Command(
+    name = "create-cpb",
+    description = ["Creates a CPB from passed in CPK archives."]
+)
+class CreateCpb : Runnable {
+
+    @Option(names = ["--cpks"], required = true, split = " ", description = ["CPKs to convert into CPB"])
+    lateinit var cpks: List<String>
+
+    @Option(names = ["--cpb-name"], required = true, description = ["Manifest attribute CPB name"])
+    lateinit var cpbName: String
+
+    @Option(names = ["--cpb-version"], required = true, description = ["Manifest attribute CPB version"])
+    lateinit var cpbVersion: String
```

Comment:
We don't need to mention "Manifest attribute" in the description. It describes how the implementation will store the values which is an implementation detail. "CPB name"/"CPB version" would be enough.

Reply:
Good point, will address.